pkg/driver/driver.go

```diff
@@ -41,11 +41,14 @@ const (
 
 const (
 	DriverName      = "ebs.csi.aws.com"
-	TopologyKey     = "topology." + DriverName + "/zone"
 	AwsPartitionKey = "topology." + DriverName + "/partition"
 	AwsAccountIDKey = "topology." + DriverName + "/account-id"
 	AwsRegionKey    = "topology." + DriverName + "/region"
```

Comment:
should we deprecate the region one as well? It seems like we only set it in the outposts case

Reply:
Hmm makes sense yeah. I didn't include it because as you said we only use it internally, but that would be more future-proof.